### ISSUE TITLE
Not all Charges are displaying on Charges Dashboard since 4/6

### ISSUE THREAD

**Issue by edgedout** [adriankumpf/teslamate#1561]
**Describe the bug**

While looking at my Grafana dashboards, I noticed that all other data has refreshed (i.e. drives, states, efficiency) however charges are not displaying since April 6.  When looking at the "Trip" dashboard, I noticed that all charges since April 6 are now displaying with null durations, costs, kw, etc)

![image](https://user-images.githubusercontent.com/33967894/115644556-6a9f4c80-a2e4-11eb-8ec9-6526abff8bfb.png)

I am currently running 1.22.0 and have signed out and signed back in using username/password per the release notes.

**Expected behavior**

Ability to see all charges and associated costs.

**How to reproduce it (as minimally and precisely as possible)**:

![image](https://user-images.githubusercontent.com/33967894/115644732-acc88e00-a2e4-11eb-9bd8-0056c27a3321.png)

**Relevant entries from the logs**

```
paste logs here
```

**Screenshots**

**Data**

**Environment**

- TeslaMate version: 1.22.0
- Type of installation: Docker
- OS TeslaMate is installed on: Ubuntu 20
- User OS & Browser: Chrome/Windows
- Others:

**Comment by edgedout** [comment]
So interesting data point.  Today I checked the "Charges" dashboard and saw that my charge from yesterday was now reflected in the dashboard but all the ones between April 6 - April 21 were still missing.  I do know that I upgraded in that period in between those dates and yesterday I adjusted the geo fence just a bit to see if that would help wake it up.

Looking at the logs I didnt see anythign that immediately stood out.  Any idea what it could have been would be appreciated.  Thanks for a great app!

![image](https://user-images.githubusercontent.com/33967894/115795781-2ff5ec80-a396-11eb-9bef-96f9b64778eb.png)

View from the "Trip Dashboard:

![image](https://user-images.githubusercontent.com/33967894/115795862-56b42300-a396-11eb-947c-d5a004bf15db.png)

![image](https://user-images.githubusercontent.com/33967894/115796012-a5fa5380-a396-11eb-9afe-0470c94c579f.png)
